osg-comanage-project-usermap.py

```diff
@@ -50,6 +55,8 @@ class Options:
     outfile = None
     authstr = None
     filtergrp = None
+    minTimeout = MINTIMEOUT
+    maxTimeout = MAXTIMEOUT
```

Comment:
```suggestion
    min_timeout = MINTIMEOUT
    max_timeout = MAXTIMEOUT
```

Attempting to align with PEP8: https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
I've incorporated this change into commit 80e513c